eng/pipelines/aggregate-reports.yml

```diff
@@ -98,6 +94,13 @@ jobs:
         Copy-Item data.js $(Build.ArtifactStagingDirectory)/staging
       displayName: 'Copy reports to artifact staging'
 
+    # Run link verification at the end as it will run no matter the pipeline passing or failing.
+    # If this task fails with an error it shouldn't prevent aggregate reports from being generated.
+    - template: ../common/pipelines/templates/steps/verify-links.yml
```

Comment:
If this step fails, the generated report doesn't get published because in the next step there's a check `condition: succeededOrFailed()` to determine if the reports should be published.

Reply:
Right, if this fails the next step will still run because it runs whether the job succeeds or fails.